src/common.ts

```diff
@@ -15,3 +25,23 @@ export interface CreateSessionOptions {
 export type GetTimestamp = {
   nanos: number; seconds: number;
 };
+
+export type Schema = string | SchemaObject;
+export type SchemaObject = IUpdateDatabaseDdlRequest;
+
+export interface RequestCallback<T> {
+  (err: ServiceError|null, response?: T|null): void;
+}
+
+//tslint:disable-next-line no-any
+export type Any = any;
+
+export type IUpdateDatabaseDdlRequest = dbAdminClient.spanner.admin.database.v1.IUpdateDatabaseDdlRequest;
+export type ITransactionOptions = spClient.spanner.v1.ITransactionOptions;
+export type ITransaction = spClient.spanner.v1.ITransaction;
+export type GeneratedTransaction = spClient.spanner.v1.Transaction;
+export type ITimestamp = spClient.protobuf.ITimestamp;
+export type IReadOnly = spClient.spanner.v1.TransactionOptions.IReadOnly;
+export type ICommitResponse = spClient.spanner.v1.ICommitResponse;
+export type IMutation = spClient.spanner.v1.IMutation;
+export type LongrunningIOperation = dbAdminClient.longrunning.IOperation;
```

Comment:
Any reason we don't just call this `IOperation`?

Reply:
Not a great reason: wanted to communicate it was Longrunning, for what it's worth 😅

Reply:
Oh! It's kind of implied, operations are only used when the API can't immediately return a response.